components/sidebar/sidebar_channel/sidebar_channel_icon/sidebar_channel_icon.tsx

```diff
@@ -8,7 +8,6 @@ import {Channel} from 'mattermost-redux/types/channels';
 type Props = {
     icon: JSX.Element | null;
     channel: Channel;
```

Comment:
Seems that we only need to know whether the channel is deleted. Should we change that to avoid unneeded rerenders?